README.md

```diff
@@ -4,62 +4,46 @@ A SBOM-centric security scanner for Kubernetes.
 
 This is still being developed. For additional details, please refer to the [RFC](docs/rfc).
 
-# Development
+# Installation
```

Comment:
This seems like a duplication of the quickstart doc. Can't we just provide a link to it inside of this section?

Reply:
removed the installation section